amnestywebsite/humanity-theme
# Remove Recipient block from the theme

Ref: https://github.com/amnestywebsite/humanity-theme/issues/144

**Steps to test**:
1. DISABLE the branding plugin
2. Edit a post
3. Check the Recipient block is no longer in the block inserter
4. ENABLE the plugin and check again
5. Make sure all functionality works as expected

Example: http://bigbite.im/v/M3G6AQ
